common/djangoapps/student/signals/receivers.py

```diff
@@ -10,7 +10,7 @@
 from django.db.models.signals import post_save, pre_save
 from django.dispatch import receiver
 
-from lms.djangoapps.courseware.toggles import REDIRECT_TO_COURSEWARE_MICROFRONTEND
+from lms.djangoapps.courseware.toggles import REDIRECT_TO_COURSEWARE_MICROFRONTEND, courseware_mfe_first_section_celebration_is_active
```

Comment:
Can probably remove the `REDIRECT_TO_COURSEWARE_MICROFRONTEND` now

Reply:
Good eye